diff --git a/node_modules/snowpack/lib/cjs/sources/local.js b/node_modules/snowpack/lib/cjs/sources/local.js
index 373e8da..d3fdecd 100644
--- a/node_modules/snowpack/lib/cjs/sources/local.js
+++ b/node_modules/snowpack/lib/cjs/sources/local.js
@@ -401,7 +401,7 @@ class PackageSourceLocal {
                 ],
             });
             const isSymlink = !entrypoint.includes(path_1.default.join('node_modules', _packageName));
-            if (isSymlink) {
+            if (isSymlink && !_packageName.startsWith("@snowstorm")) {
                 return;
             }
         }
