diff --git a/node_modules/snowpack/lib/index.js b/node_modules/snowpack/lib/index.js
index ac926b4..a0760a1 100644
--- a/node_modules/snowpack/lib/index.js
+++ b/node_modules/snowpack/lib/index.js
@@ -123181,7 +123181,7 @@ class PackageSourceLocal {
                 ],
             });
             const isSymlink = !entrypoint.includes(path_1.default.join('node_modules', _packageName));
-            if (isSymlink) {
+            if (isSymlink && !_packageName.startsWith("@snowstorm")) {
                 return;
             }
         }
